stonesoup/initiator/simple.py

```diff
@@ -110,11 +113,12 @@ def initiate(self, detections, **kwargs):
                 model_matrix.T @ np.ones((model_matrix.shape[0], 1)))
             prior_state_vector[mapped_dimensions, :] = 0
             prior_covar[mapped_dimensions, :] = 0
-
+            C0 = inv_model_matrix @ model_covar @ inv_model_matrix.T
```

Comment:
The last term has changed from `model_matrix.astype(bool)` to `inv_model_matrix.T`. Is that right? I can see it's changed the result in [the test below](https://github.com/dstl/Stone-Soup/pull/266/files#diff-c0310c8dfe4bac1ad2388ecabd89e3f7R182)

Reply:
Yes. This is the main change that was causing the trackers to fail. If you look at the test, I added in assertions that check that the state covariance matrix is consistent with the measurement noise. 
If we assume that R<sub>y</sub> = H R<sub>x</sub> H<sup>T</sup> then R<sub>x</sub> is approximated by  G R<sub>y</sub> G<sup>T</sup>, where G is the psuedo-inverse of H.